sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -171,6 +171,7 @@ class CosmosClient:  # pylint: disable=client-accepts-api-version-keyword
         level (to log all requests) or at a single request level. Requests will be logged at INFO level.
     :keyword bool no_response_on_write: Indicates whether service should be instructed to skip sending 
         response payloads for write operations on items by default unless specified differently per operation.
+    :keyword int throughput_bucket: The desired throughput bucket for the client
```

Comment:
The readme mentioned that the feature is in public preview? If that's the case, then everywhere this parameter is documented, it needs to be marked as provisional.